android/src/main/java/io/parity/signer/screens/scan/camera/CameraView.kt

```diff
@@ -27,6 +32,10 @@ internal fun CameraViewInternal(viewModel: CameraViewModel) {
 	val cameraProviderFuture =
 		remember { ProcessCameraProvider.getInstance(context) }
 	val coroutineScope = rememberCoroutineScope()
+	val configuration = LocalConfiguration.current
+val backgroundExecutor = remember {
```

Comment:
```suggestion
    val backgroundExecutor = remember {
```
But seems like something that linter / formatter should catch

Reply:
we don't have automated formatter, formatted.